src/v0/util/index.js

```diff
@@ -2025,6 +2025,11 @@ const getAuthErrCategoryFromStCode = (status) => {
   return '';
 };
 
+const validateEventType = event => {
+  if(!event || typeof event !== "string"){
+    throw new InstrumentationError("Event is a required field and should be a string");
```

Comment:
After throwing this, are we getting related statTags ? Can we once validate it ?

Reply:
Yes we are getting the stat tags as well. SS fro ref:
![Uploading Screenshot 2023-10-05 at 12.06.55 PM.png…]()